packages/integration-tests/package.json

```diff
@@ -12,13 +12,14 @@
   "publishConfig": {
     "access": "public"
   },
-  "homepage": "https://github.com/nasa/cumulus/tree/master/packages/integration-tests",
+  "homepage": "https://github.com/nasa/cumulus/tree/master/packages/integration-tests/README.md",
   "repository": {
     "type": "git",
     "url": "https://github.com/nasa/cumulus"
   },
   "scripts": {
     "build": "rm -rf dist && mkdir dist && webpack",
+    "build-docs": "jsdoc2md --template README.hbs Collections.js Executions.js Granules.js Providers.js Rules.js > README.md",
```

Comment:
Genuine question: What do you think about generating the docs to `docs/API.md`? I ask because I started doing this in other packages since we are linting the markdown in our package READMEs, but not in subdirectories of the packages. And I worry that jsdoc2md may produce invaild markdown

Reply:
The big advantage of having the API docs in the readme is that they also show up when you view the package on npm's website. I checked just now and confirmed that, at least so far, the docs being generated by `jsdoc2md` do pass our markdown linter.